docs/src/components/component/breadcrumb-nav/bc-app-view/bc-app-view--app-closed.html

```diff
@@ -0,0 +1,62 @@
+<nav class="bg-white p4 mn2">
+  <div class="row">
+    <div class="col-fit h8 row-end">
+      <div class="h6 row-center-x">
+        {% render '@icon-square--home-active-large'%}
```

Comment:
I think I just don't know how to to that in the templates — I agree tho, that the icon size could/should be defined in its parent component. This prototype doesn't use subcomponents, tho (outside of the icon component).

Unless I misunderstand the comment.